### PR TITLE
Projects: Only display 'Submit Application' before deadline

### PR DESCRIPTION
**Pull request by PeterMPhillips**
Screenshot from 2019-12-19 16-20-51

### CODE REVIEW

**Comment by stellarmagnet** [comment]
@PeterMPhillips the title of this PR seems incorrect, right?

**Comment by PeterMPhillips** [comment]
@stellarmagnet I'm not sure what the issue is with the title. We don't display 'Submit application' in the context menu if the deadline has already passed

File: apps/projects/app/components/Shared/BountyContextMenu.js [hunk 1]
```diff
@@ -74,12 +74,14 @@ const BountyContextMenu = ({ issue }) => {
...
-            <IconFile color={`${theme.surfaceContent}`} />
-            <ActionLabel>
-              Submit application
-            </ActionLabel>
-          </Item>
+          {!pastDeadline && (
+            <Item onClick={() => requestAssignment(issue)}>
+              <IconFile color={`${theme.surfaceContent}`} />
+              <ActionLabel>
+                Submit Application
```
Comment (chadoh) [inline apps/projects/app/components/Shared/BountyContextMenu.js]: I believe we want to keep this sentence-case. See #1682 & #1669

Comment (PeterMPhillips) [reply apps/projects/app/components/Shared/BountyContextMenu.js]: Done

**Comment by stellarmagnet** [comment]
@PeterMPhillips what is the state of this? Should we get it merged?

**Pull request merged**